README.md

```diff
@@ -1,2 +1,29 @@
 # nf-agora
-Nextflow workflow for agora-data-tools data processing
+
+## Intro
+
+This repository contains a Nextflow Pipeline that wraps agora-data-tools and therefore the Agora ETL process. It is deployed on Sage's Nextflow Tower Instance, making it simple to trigger and monitor runs. 
+
+## Running the pipeline on Nextflow Tower
+
+### Getting Oriented in Nextflow Tower
+
+1. Navigate to https://tower.sagebionetworks.org/login.
+2. Click “Sign in with Google” and enter your Sage credentials.
+3. In the top left corner, you should see “Sage-Bionetworks/agora-project”. This indicates that you are in the workspace for Agora in our Nextflow Tower instance.
+4. If you are not in the correct workspace, click on the drop-down and select “agora-project”.
+
+### Launching the Pipeline
+
+1. In the “Launchpad” tab, click on the tile labeled “nf-agora”.
```

Comment:
We may want to add a section linking out to user secrets. 

What is the secret you have configured here? Your synapse user? We will have to add the service user secrets in the workspace secrets eventually.